src/api/domain/exchanges-api.ts

```diff
@@ -308,14 +309,15 @@ export const ExchangesApiFp = function(configuration?: Configuration) {
          * Accepts a transfer request and updates the transfer_status after processing transfer (offramp)
          * @summary Accepts a transfer request and updates the transfer_status after processing transfer (offramp)
          * @param {string} id Transaction ID
-         * @param {string} xImxEthAddress eth address
-         * @param {string} xImxEthSignature eth signature
          * @param {CreateTransferRequestV1} createTransferRequest Create a transfer
+         * @param {string} [xImxEthAddress] eth address
+         * @param {string} [xImxEthSignature] eth signature
+         * @param {string} [authorization] Authorization header
          * @param {*} [options] Override http request option.
          * @throws {RequiredError}
          */
-        async createExchangeTransfer(id: string, xImxEthAddress: string, xImxEthSignature: string, createTransferRequest: CreateTransferRequestV1, options?: AxiosRequestConfig): Promise<(axios?: AxiosInstance, basePath?: string) => AxiosPromise<CreateTransferResponseV1>> {
-            const localVarAxiosArgs = await localVarAxiosParamCreator.createExchangeTransfer(id, xImxEthAddress, xImxEthSignature, createTransferRequest, options);
+        async createExchangeTransfer(id: string, createTransferRequest: CreateTransferRequestV1, xImxEthAddress?: string, xImxEthSignature?: string, authorization?: string, options?: AxiosRequestConfig): Promise<(axios?: AxiosInstance, basePath?: string) => AxiosPromise<CreateTransferResponseV1>> {
```

Comment:
Here is the entry point. The authorization is optional, and doesn't have a breaking change.

Reply:
This could be a breaking change because this function is not using an object for parameters we rely on the order of the function parameters, this is adding the `authorization` param now before the `options`, even with it being optional this could end up being a breaking change if this is the case?

Reply:
Sorry, I pointed it to the wrong place. https://github.com/immutable/imx-core-sdk/pull/294/files#diff-260b1ddfcb91599047eb8ecc7002866d68872c7d81e0f7560375f85fc1639872R630